idaes/models_extra/co2_capture_and_utilization/unit_models/tests/test_membrane_1d.py

```diff
@@ -66,17 +68,44 @@ def test_config():
         dynamic=False,
         sweep_flow=True,
         flow_type=MembraneFlowPattern.COUNTERCURRENT,
-        property_package=m.fs.properties,
+        feed_side={"property_package": m.fs.properties},
+        sweep_side={"property_package": m.fs.properties},
+    )
+
+    # Check unit config arguments
+    print("=====================================================")
```

Comment:
You should remove these print statements for the tests.

Reply:
removed